src/Reprocessing_Draw.rei

```diff
@@ -13,7 +63,7 @@
 let translate: (~x: float, ~y: float, Reprocessing_Types.Types.glEnvT) => unit;
 
 
-/*** Rotates the amount specified by the angle parameter. Angles must be
+/** Rotates the amount specified by the angle parameter. Angles must be
```

Comment:
refmt's parser only accepts doccomments that start with 2 stars. these comments were being ignored